Readme.md

```diff
@@ -1,19 +1,30 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128594795/14.2.7%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T237508)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [BusinessObject.cs](CS/EF/CustomFileDataEF/CustomFileDataEF.Module/MyModels/BusinessObject.cs)
-* **[InplaceFileData.cs](CS/EF/CustomFileDataEF/CustomFileDataEF.Module/MyModels/InplaceFileData.cs)**
-<!-- default file list end -->
-# How to use the File Attachment Module with a legacy database
+# XAF - How to use the File Attachment Module with a legacy database
+
+The [File Attachments](https://docs.devexpress.com/eXpressAppFramework/112781/document-management/file-attachments-module) module ships with the capability to easily implement a file upload and download functionality in your application. However, the built-in `FileData` class is persistent and therefore requires a separate table in the database to store file contents and names. These specifics make this class useless for XAF applications that must work with a legacy database where a file is stored in a BLOB field of the table with other fields of the same object and in a plain form. Also, the same object may contain multiple BLOB fields for different files. 
+
+
+## Implementation Details
 
+To handle this and other various scenarios, you can create a non-persistent class that implements the `IFileData` interface. This example demonstrates a possible `IFileData` implementation in a non-persistent class that actually passes data to and from the owner persistent object, where the BLOB (`byte[]`) property is declared.
```

Comment:
- You do not 'handle' a scenario
- `and other various scenarios` I'm not sure why you use 'various' here
- `that actually passes` What is the difference between "that passes" and "that actually passes"? Does the former mean it does not pass data? 

Rewrite

Reply:
Hi Dirk. Does this variant work?

```suggestion
To execute this and similar scenarios, you can create a non-persistent class that implements the `IFileData` interface. This example demonstrates a possible `IFileData` implementation in a non-persistent class that passes data to/from the owner persistent object where the BLOB (`byte[]`) property is declared.
```

Reply:
Hi Irina, yes, it looks fine